src/extra/CMakeLists.txt

```diff
@@ -9,11 +9,15 @@ set(extra_SOURCES
   gfx/shader/translator.cpp
```

Comment:
Prefer target_sources (https://cmake.org/cmake/help/latest/command/target_sources.html) instead of SET.
Have a look at Modern CMake

Reply:
This shouldn't be confused with global sources, this is just a variable passed to `add_library` containing the source files.
I just like to group them in a separate variable usually for clarity